examples/landmark_uncertainty/README.md

```diff
@@ -0,0 +1,34 @@
+# Uncertainty Estimation in Landmark Localisation
```

Comment:
See the first line under https://github.com/pykale/pykale/blob/main/.github/CONTRIBUTING.md#medium-involvements-contributors, we use US English spelling in PyKale.